README.md

```diff
@@ -1,6 +1,6 @@
 # Metarhia Common Library
 
-![CI Status Badge](https://github.com/metarhia/common/workflows/Tests/badge.svg?branch=master)
+[![CI Status](https://github.com/metarhia/common/workflows/Testing%20CI/badge.svg)](https://github.com/metarhia/common/actions?query=workflow%3A%22Testing+CI%22+branch%3Amaster)
```

Comment:
This is a wrong workflow name for this repo, it should be
`https://github.com/metarhia/common/actions?query=workflow%3A%22Tests%22+branch%3Amaster`